frontend/package.json

```diff
@@ -29,8 +28,9 @@
         "test": "echo No tests... yet",
         "fmt": "prettier \"css/**/*\" \"ts/**/*\" \"*.json\" \"*.js\"",
         "dev": "parcel watch \"html/**/*.html\"",
-        "build": "mkdir -p ../embed/templates && rm -rf ../embed/templates/* && parcel build --no-source-maps --no-cache \"html/**/*.html\"",
-        "build:windows": "pwsh --command ../scripts/windows/frontend_build.ps1"
+        "build": "mkdir -p ../embed/templates && rm -rf ../embed/templates/* && yarn build:parcel",
+        "build:parcel": "parcel build --no-source-maps --no-cache \"html/**/*.html\"",
+        "build:windows": "pwsh --command ../scripts/windows/frontend_cleanup.ps1 && yarn build:parcel"
```

Comment:
Can pwsh handle &&?

Reply:
@natsukagami defo, I've always used that.